package.json

```diff
@@ -35,6 +35,7 @@
     "dotenv": "^10.0.0",
     "ethers": "^5.5.2",
     "hardhat": "^2.7.1",
+    "hardhat-contract-sizer": "^2.6.1",
```

Comment:
Comment by @h4x0r31337:
> Please pin the version and do not use `^`

Reply:
We don't need the carpet to pin the exact version. Other packages are currently denoted with `^` and show no problem. Moreover, the `yarn.lock` file is committed along with the commit, and contains the info of the exact version to be installed. One can obtain the same by running `yarn --frozen-lockfile`.